src/chi/utilities/close.scss

```diff
@@ -0,0 +1,35 @@
+@import 'mixins';
```

Comment:
@davidojedacl utilities/close.scss makes no sense in utilities anymore as we have moved this component to buttons. Put this scss in buttons.scss below .-float class.